common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/RouteSearchTabConstants.kt

```diff
@@ -0,0 +1,36 @@
+package com.trafi.ui.component.internal
+
+import com.trafi.ui.theme.internal.*
+import com.trafi.ui.theme.internal.BasicFontWeight.Bold
+import com.trafi.ui.theme.internal.BasicFontWeight.Normal
+import com.trafi.ui.theme.internal.dp
+
+class RouteSearchTabConstants (theme: CurrentTheme) {
+    val contentColorPrimary = theme.colorPalette.onSurface
+    val contentColorSecondaryOnLight = theme.colorPalette.grayScale.gray600
+    val contentColorSecondaryOnDark = theme.colorPalette.grayScale.gray400
+
+    val backgroundColorDefaultOnLight = theme.colorPalette.grayScale.gray100
+    val backgroundColorDefaultOnDark = theme.colorPalette.grayScale.gray800
```

Comment:
gray scale is automatically inverted from theme, you only need a single constant